.github/workflows/build_and_upload_artifact.yml

```diff
@@ -0,0 +1,33 @@
+name: Build and generate artifacts for Bahmni Core
+
+on:
+  push:
+    branches: 
+      - master
+      - BAH-*
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+        server-id: github
```

Comment:
Setting up Maven environments for use here. This is done with setup-java because otherwise running setup-java again overwrites the settings.xml